networkx/algorithms/tests/test_walks.py

```diff
@@ -0,0 +1,103 @@
+"""Unit tests for the :mod:`networkx.algorithms.walks` module."""
+
+from functools import partial
+
+from networkx import (
+    DiGraph,
+    all_pairs_number_of_walks,
+    cycle_graph,
+    empty_graph,
+    number_of_walks,
+    relabel_nodes,
+    single_source_number_of_walks,
+)
```

Comment:
I would say you should just use e.g. `nx.single_source_number_of_walks` instead of importing each individually.  Same for `pairwise`. But that's really just a style issue.